vulnfeeds/cmd/ids/main.go

```diff
@@ -141,11 +147,22 @@ func assignIDs(prefix, dir string) error {
 		return fmt.Errorf("failed to walk: %w", err)
 	}
 
+	if len(unassigned) == 0 {
+		fmt.Printf("Nothing to allocate")
+		return nil
+	}
+
 	fmt.Printf("Assigning IDs using detected maximums: %v\n", yearCounters)
 	for _, path := range unassigned {
 		if err := assignID(path, prefix, yearCounters, defaultYear); err != nil {
 			return fmt.Errorf("failed to assign ID: %w", err)
 		}
 	}
-	return nil
+
+	b := make([]byte, conflictMarkerSize)
+	if _, err := rand.Read(b); err != nil {
+		return fmt.Errorf("failed to generate random string: %w", err)
+	}
+
+	return ioutil.WriteFile(filepath.Join(dir, conflictFile), []byte(hex.EncodeToString(b)), 0644)
```

Comment:
Where is this file used ?

Reply:
This will be picked up by "git add/commit". If there are conflicting assignments, conflicts can be detected if this file has a merge conflict.